forms/jqm/subformcollapse.js

```diff
@@ -35,13 +43,17 @@ define(function (require) {
       this.$summary = this.$collapsible.children('h3')
         .children('.bm-subform__bm-summary');
 
-      this.$collapsible.collapsible();
+      this.$collapsible.collapsible({
+        collapsed: this.model.get('collapsed')
+      });
 
       this.$el.attr(
         'data-record-index',
         this.model.parentElement.attributes.forms.indexOf(this.model)
       );
 
+      this.$toggleTrigger = $('.ui-collapsible-heading-toggle', this.$el);
```

Comment:
Does this line work as expected when collapsible subforms are nested within collapsible subforms?